// protobuf definition for firewall log

syntax = "proto3";

package fwlog;

message FWEvent {
    int32 action = 1;
    int32 ipVer = 2;
    uint32 sipV4 = 3;
    uint32 dipV4 = 4;
    uint32 sport = 5;
    uint32 dport = 6;
    uint32 ipProt = 7;
    uint32 direction = 8;
    uint64 sipv61 = 9;
    uint64 sipv62 = 10;
    uint64 dipv61 = 11;
    uint64 dipv62 = 12;
}
